martin/src/srv/server.rs

```diff
@@ -105,7 +105,8 @@ async fn get_health() -> impl Responder {
     "/catalog",
     method = "GET",
     method = "HEAD",
-    wrap = "middleware::Compress::default()"
+    wrap = "middleware::Compress::default()",
+    wrap = "actix_middleware_etag::Etag::default()"
```

Comment:
please add a `use actix_middleware_etag::Etag;` everywhere, and also a trailing comma for all - to avoid churn

Reply:
> also a trailing comma for all - to avoid churn

tell that to `cargo fmt`.
I don't see an option for this here too: https://rust-lang.github.io/rustfmt/